DependencyInjection/Configuration.php

```diff
@@ -34,6 +34,7 @@ public function getConfigTreeBuilder()
                 ->scalarNode('title')->cannotBeEmpty()->isRequired()->info('The title of the API.')->end()
                 ->scalarNode('description')->cannotBeEmpty()->isRequired()->info('The description of the API.')->end()
                 ->scalarNode('cache')->defaultFalse()->info('The caching service to use. Set to "dunglas_api.mapping.cache.apc" to enable APC metadata caching.')->end()
+                ->booleanNode('enable_doctrine_orm')->defaultValue(true)->info('Enable the Doctrine ORM integration.')->end()
```

Comment:
Here you can test if the a class from the Doctrine bridge exist to set the default value.